3600/3/app/controllers/restaurant_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require_relative '../../helpers/comment_helper.rb'
+
+class RestaurantController < Sinatra::Base
```

Comment:
Inherit it from AppController

Reply:
Done

Reply:
Inherited

Reply:
![Screenshot from 2019-08-02 22-52-38](https://user-images.githubusercontent.com/46228563/62395207-3d42c600-b578-11e9-85b0-d9c7e019ecc5.png)